src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const SHORT_TERM_SAIL = 20;
+  const LONG_TERM_SAIL = 50;
+  const CAR_RENT = 40;
+  const LONG_TERM_RENT = 7;
+  const SHORT_TERM_RENT = 3;
+  const TOTAL_RENT = CAR_RENT * days;
```

Comment:
This line calculates the total rent cost, which is a constant operation and could be placed directly in the return statement to avoid unnecessary variable assignment.